corpus/declarations.txt

```diff
@@ -272,8 +272,8 @@ Variadic function declarations
 package main
 
 func f1(a ...*int) {}
-func f2(...int) {}
-func f3(a, ...bool) {}
```

Comment:
The declaration `func f3(a, ... bool) {}` turns out not to be valid Go.

Reply:
Oh, good point. I think the reason that the reason I included these function declarations without parameter names is that you *can* write these as part of an `interface` declaration (in which only the types are important). For example,

```go
type Person interface {
  func AddFriends(...Person) error
}
```

I didn't realize that you could *only* omit the parameter names in an interface declaration though. Thanks for pointing out that these aren't valid.

Would you mind adding back these examples, but just inside of an `interface`, so that they actually represent valid Go code? This test might not be the place for that though; maybe they should go with the other test(s) about interface declarations?

Reply:
@maxbrunsfeld, @aymannadeem and I added test cases for `variadic_parameter_declaration` for `Interface` and `Function` types in https://github.com/tree-sitter/tree-sitter-go/pull/11/commits/23c3e32bfe1a2b15766b70d3b80db9fc8232bf7e.